WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -1278,6 +1322,23 @@ public void onAvailabilityChecked(OnAvailabilityChecked event) {
         }
     }
 
+    private void handleUsernameAvailabilityEvent(OnAvailabilityChecked event) {
```

Comment:
It would be nice to have a comment header explaining what this routine is doing.